### PR TITLE
added data to 2009A%26A...499..723A/tev-000132.yaml

### PR DESCRIPTION
**Pull request by pdeiml**
Vergleich bitte mal das Paper mit meinem Eintrag bei pos: ra und dec. Nicht dass ich die beiden vertauscht habe.

### CODE REVIEW

File: input/data/2009/2009A%26A...499..723A/tev-000132.yaml [hunk 1]
```diff
@@ -1,4 +1,26 @@
 source_id: 132
 reference_id: 2009A&A...499..723A
 
-# TODO: add data
+data:
+  # TODO: No livetime found in the paper
+  livetime:
```
Comment (cdeil) [inline input/data/2009/2009A%26A...499..723A/tev-000132.yaml]: livetime is 27 hours.

Section 2:
> The total dead-time corrected, quality and offset selected (2.5 ) data set amounts to 27 hours

File: input/data/2009/2009A%26A...499..723A/tev-000132.yaml [hunk 2]
```diff
@@ -1,4 +1,26 @@
...
+  significance: 12.1
+  excess: 689
+
+pos:
+  ra: {val: 40d23m9.2s, err: 2.4m, err_sys: 20s}
+  dec: {val: 0d47m10.1s, err: 2.4m, err_sys: 20s}
```
Comment (cdeil) [inline input/data/2009/2009A%26A...499..723A/tev-000132.yaml]: Missing minus sign: `val: -0d47m10.1s`

File: input/data/2009/2009A%26A...499..723A/tev-000132.yaml [hunk 3]
```diff
@@ -1,4 +1,26 @@
...
+
+morph:
+  type: gauss
+
+  # TODO: sigma and sigma2 have assymmetric errors: -0.03, +0.04, no 'pa' value found
+  sigma: {val: 0.34, err: 0.04}
+  sigma2: {val: 0.34, err: 0.03}
```
Comment (cdeil) [inline input/data/2009/2009A%26A...499..723A/tev-000132.yaml]: Remove sigma2. A symmetric, not elongated Gaussian was fitted.

File: input/data/2009/2009A%26A...499..723A/tev-000132.yaml [hunk 4]
```diff
@@ -1,4 +1,26 @@
...
+  excess: 689
+
+pos:
+  ra: {val: 40d23m9.2s, err: 2.4m, err_sys: 20s}
+  dec: {val: 0d47m10.1s, err: 2.4m, err_sys: 20s}
+
+morph:
+  type: gauss
+
+  # TODO: sigma and sigma2 have assymmetric errors: -0.03, +0.04, no 'pa' value found
```
Comment (cdeil) [inline input/data/2009/2009A%26A...499..723A/tev-000132.yaml]: Remove TODO comment.
You can keep a comment that the error given in the paper is asymmetrical, and that the down error is 0.03 deg.

File: input/data/2009/2009A%26A...499..723A/tev-000132.yaml (hunk 2 above)
Comment (cdeil) [inline input/data/2009/2009A%26A...499..723A/tev-000132.yaml]: In this case the position is given in the paper as l / b, i.e. in Galactic coordinates. (not in RA / DEC).
As mentioned at https://github.com/gammapy/gamma-cat/blob/master/input/schemas/dataset_source_info.schema.yaml#L45 for those cases we should use the key names `glon:` and `glat:`

File: input/data/2009/2009A%26A...499..723A/tev-000132.yaml (hunk 4 above)
Comment (pdeiml) [inline input/data/2009/2009A%26A...499..723A/tev-000132.yaml]: What about the "pa" value which I didn't found. Does it have to be given or is it optional?
If it has to be given, then we should think about some default value.

Comment (cdeil) [inline input/data/2009/2009A%26A...499..723A/tev-000132.yaml]: Usually when values aren't given, the idea is to just not put the key at all in the input files.
So no default needed for `pa` and `sigma2` here, just omit it completely in the input file.

OK?

**Comment by pdeiml** [comment]
Rechecked the errors.
Ready to merge.

**Comment by cdeil** [comment]
Thanks!

**Pull request merged**